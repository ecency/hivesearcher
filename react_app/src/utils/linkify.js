export default (author, permlink) => (`https://steemit.com/@${author}/${permlink}`);