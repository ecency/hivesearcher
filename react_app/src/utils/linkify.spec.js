/* eslint-disable */
import linkify from './linkify';

describe('Linkify', () => {
    it('(1) ', () => {
        expect(linkify("foo", "fancy-post-title-001")).toMatchSnapshot();
    });

});