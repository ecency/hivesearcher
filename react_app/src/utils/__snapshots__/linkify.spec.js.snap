// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Linkify (1)  1`] = `"https://steemit.com/@foo/fancy-post-title-001"`;
